soc/renesas/smartbond/da1469x/soc.c

```diff
@@ -173,7 +177,7 @@ static int renesas_da1469x_init(void)
 	da1469x_trimv_init_from_otp();
 
 	da1469x_pd_init();
-	da1469x_pd_acquire(MCU_PD_DOMAIN_SYS);
```

Comment:
Based on my comment above, here is the right place to acquire PD_SYS.

Reply:
Correct. I moved `da1469x_pdc_reset();` after this.